_transcripts/039-rust-1.54-1.55.md

```diff
@@ -0,0 +1,718 @@
+---
+episode: _episodes/039-rust-1.54-1.55.md
+---
+
+{%- include util.html -%}
+[episode]: {{episode.url}}
+
+__Jon Gjengset__: Hello Ben, welcome back.
+
+__Ben Striegel__: It's good to be back, Jon. How are you doing?
+
+__Jon__: I'm pretty excited, you know. We get to do two versions of Rust. It's
+not every day.
+
+__Ben__: Not every day, only once every three months, so four times a year, four
+days a year.
+
+__Jon__: Definitely not a common occurrence. So I'm in a festive mood, I'm ready
+to celebrate 1.54 and 1.55.
+
+__Ben__: Yeah. One of these releases is pretty short, so I think this first one
+here, 1.54, won't take us long to get through. Do you wanna kick us off there,
+Jon?
+
+__Jon__: I feel like we say this, and then we always end up spending, like, a
+ton of time on some really weird detail.
+
+__Ben__: No, I promise. This time it is a pretty small release.
+
+__Jon__: All right, let's try. So, 1.54 starts out with "attributes can invoke
+function-like macros." And this one is, on the surface, pretty simple. If you
+have an attribute macro, so this is something that starts with a square bracket
+and then an optional exclamation mark and then square brackets. If you have one
```

Comment:
I'm tempted to add a correction here:
> ... starts with a square bracket *(editor's note: starts with a `#`)*

(at 01:04)

Reply:
Seems reasonable to me!